Library/Homebrew/test/utils/shell_spec.rb

```diff
@@ -0,0 +1,68 @@
+require "utils/shell"
+
+describe Utils::Shell do
```

Comment:
I think it would be nicer to set a subject rather than calling all the methods on `described_class`:

```ruby
subject { Object.new.extend(described_class) }
```